src/components/Application/Application.jsx

```diff
@@ -1,32 +1,62 @@
-import React, { PropTypes } from 'react';
+import React, { PropTypes, Component } from 'react';
 import { connect } from 'react-redux';
 
+import { ipcRenderer } from 'electron';
+
 import OperatorPanel from '../OperatorPanel/OperatorPanel.jsx';
 import ClientsPanel from '../ClientsPanel/ClientsPanel.jsx';
 import MessagePanel from '../MessagePanel/MessagePanel.jsx';
-import Settings from '../Settings/Settings.jsx';
+import SettingsPanel from '../SettingsPanel/SettingsPanel.jsx';
+
+import { setConfig } from '../../store/Chat/actions.js';
+
 import './Application.css';
 
-const Application = (props) => {
-  const renderMainView = () => (
-    props.settingsOpen ? <Settings /> :
+class Application extends Component {
+  constructor (props) {
+    super(props);
+
+    // Setup our IPC listener
+    ipcRenderer.on('config', this.updateConfig());
+    ipcRenderer.send('init-config');
+  }
 
+  updateConfig () {
+    const { updateConfig } = this.props;
+
+    return function (event, config) {
+      // Send config to Redux state
+      updateConfig(config);
+    };
+  }
```

Comment:
This is confusing me a bit...

you have a class method called `updateConfig`...  but you also have a function on the `dispatch` object on line 69. I think this is similar to how the redux action creators were dispatched inside the mnml client. 

basically, the function that is dispatched is setup in the `mapDispatchToProps` so that on line 20, you should only have to do:

`ipcRenderer.on('config', this.props.updateConfig()` and you shouldn't have to define the `updateConfig` class method. So I guess right now you have two instances of updateConfig, but they are differently used... but named the same ? I'm not sure if I'm missing something... hope that makes sense.